scripts/bootstrap-vault.sh

```diff
@@ -0,0 +1,51 @@
+# Init vault and save keys
```

Comment:
Is this script allowed to fail? Consider:
```suggestion
set -euxo pipefail
# Init vault and save keys
```
An explanation:
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
No, it shouldn't be allowed to fail.